### PR TITLE
Added pss panic to known issues for 1.7.0 and 1.7.1 and troubleshooting also indicated that this is fixed in release 1.7.2

### PR DESCRIPTION
**Pull request by bsoroushian**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?
1.7.1, 1.7.0, main

It's best to PR to the most recent relevant branch and leave all the cherry-picking to the
writer, except where earlier branches require factual changes to the content.
For more information about the branches, see https://github.com/pivotal/docs-tap#branches

### CODE REVIEW

File: release-notes.hbs.md [hunk 1]
```diff
@@ -45,6 +45,10 @@ The following issues, listed by component and area, are resolved in this release
 
 - Resolved issue description.
 
+#### <a id='1-7-2-COMPONENT-NAME-ri'></a> v1.7.2 Resolved issues: Supply Chain Security Tools - Scan and Sacn 2.0
```
Comment (ryan-a-baker) [inline release-notes.hbs.md:48]:
```suggestion
#### <a id='1-7-2-scst-scan-ri'></a> v1.7.2 Resolved issues: Supply Chain Security Tools - Scan an Scan 2.0
```

File: release-notes.hbs.md [hunk 2]
```diff
@@ -45,6 +45,10 @@ The following issues, listed by component and area, are resolved in this release
...
+
+- Controller panic when `container.SecurityContext` is provided but either of fields `Capabilities` or `SeccompProfile` are left emptty is addressed.
```
Comment (ryan-a-baker) [inline release-notes.hbs.md:50]:
```suggestion
- Scan Controller panic when `container.SecurityContext` is configured but not all fields are provided such as  `Capabilities` or `SeccompProfile` has been resolved.
```

File: scst-scan/troubleshoot-scan.hbs.md [hunk 3]
```diff
@@ -563,3 +563,32 @@ restricted Pod Security Standard.
 
 1. Apply the new template to your cluster.
 1. Update the `tap-values.yaml` file to use the new template in your workloads.
+
+### <a id="pss-panic"></a> Panic when container.SecurityContext is provided but some fields left empty
+
+**Symptom:**
+
+When `container.SecurityContext` is provided but either of fileds `Capabilities` or `SeccompProfile` are left empty like below, the controller for for TAP 1.7.0 and 1.7.1 will panic.
```
Comment (ryan-a-baker) [inline scst-scan/troubleshoot-scan.hbs.md:571]:
```suggestion
When `container.SecurityContext` is provided but either the field `Capabilities` or `SeccompProfile` is left empty like below, the controller will panic.
```

**Comment by kcoriordan** [comment]
@bsoroushian 
Does this need to be cherry picked to 1.8 (main)?
Do I need to add a known issue or resolved issue to main for this?

**Comment by bsoroushian** [comment]
@kcoriordan after discussion we decided that this change not be cherry-picked into `main`.

**Pull request merged**